app/assets/stylesheets/application/modules/_sections_slabs_cards.scss

```
@@ -65,39 +76,41 @@
   margin-bottom: 9px;
   clear: left;
 }
+
 .health__team-member--type i {
   font-size: 2.5em;
   vertical-align: middle;
 }
-.health__team-member--name
-{
+
+.health__team-member--name {
   margin-top: 2em;
 }
+
 .health__team-member__individual {
   min-height: 180px;
 }
+
 .health__team-member__action,
-.health-careplan__goal-action
-{
+.health-careplan__goal-action {
   display: inline-block;
   padding-left: 12px;
 }
+
 .health__team-member__action a,
-.health-careplan__goal-action a
-{
-  color: $cob-sg-blue-006;
+.health-careplan__goal-action a {
+  color: var(--brand-primary-l);
 }
+
 .health__team-member__action a:hover,
 .health__team-member__action a:active,
 .health__team-member__action a:focus,
 .health-careplan__goal-action a:hover,
 .health-careplan__goal-action a:active,
-.health-careplan__goal-action a:focus
- {
-  color: darken($cob-sg-blue-006, 20);
+.health-careplan__goal-action a:focus {
+  color: var(--brand-primary-l);
```

this seems like the same color as the default state, so we the hover state no longer has visual differentiation.

```suggestion
  color: var(--brand-primary-l);
  filter: brightness(80%);